closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -128,6 +133,7 @@ public String[] run() {
         isFIPSSupported = isNSSSupported;
 
         userEnabledFIPS = Boolean.parseBoolean(props[0]);
+        allowSetSecurityProps = Boolean.parseBoolean(props[4]);
```

Comment:
The field was renamed `allowSetProperties`.

Reply:
Sorry about that. Thought I refactored there. Fixed.